docs/customization.rst

```diff
@@ -9,6 +9,32 @@ Sortables
 
 The steps for enabling drag-and-drop sorting functionality is identical to Grappelli’s (even with the vanilla Django admin), so the `Grappelli documentation <http://django-grappelli.readthedocs.org/en/latest/customization.html#inline-sortables>`_ for this feature is the best reference. This is equally true of sortables in Django Suit—do not follow Django Suit’s instructions for sortables, as they will not work. The `other features of Grappelli <http://django-grappelli.readthedocs.org/en/latest/customization.html>`_ have not been ported to work with vanilla Django, but in principle they should all still work with nested inlines using django-nested-admin if Grappelli is installed. If you run into any difficulty with this, please `create an issue <https://github.com/theatlantic/django-nested-admin/issues>`_ on this project’s Github.
 
+If you are using vanilla Django and want to be able to sort an inline but also want to hide the field you're soring with then you can use the mixin ``nested_admin.forms.SortableHiddenMixin``. Note that this mixin sets ``sortable_field_name`` to ``position`` by default and must be inherited prior to ``admin.NestedStackedInline`` or ``admin.NestedTabularInline``.
+
+.. code-block:: python
+
+    # An example admin.py for a Table of Contents app
+
+    from django.contrib import admin
+	from nested_admin.forms import SortableHiddenMixin
```

Comment:
This should probably be indented a bit more?

Reply:
Hi - I fixed this typo and updated from upstream master, but I'm seeing the below issues from Codecov and TravisCI from other code in master. What's the best way to resolve those issues and help get this PR in place? Thanks!